webui/react/src/pages/F_ExpList/glide-table/TableActionBar.tsx

```diff
@@ -156,9 +156,9 @@ const TableActionBar: React.FC<Props> = ({
 
   const selectedExperiments = useMemo(
     () =>
-      Array.from(selectedExperimentIds).flatMap((id) =>
-        id in experimentMap ? [experimentMap[id]] : [],
-      ),
+      Array.from(selectedExperimentIds).flatMap((id) => [
+        { id, unmanaged: id in experimentMap ? experimentMap[id].unmanaged : false },
+      ]),
```

Comment:
question: why not just return experiments? imo, returning `experiments` is more meaningful even if modal doesnt use the info for now

Reply:
Also, if backend ignore the unmanaged experiments, can we just send all experiments anyways? then we can remove `selectedExperiments` code?

Reply:
Does this commit help? We can just pass IDs and show a modal if we know we selected an unmanaged experiment.  https://github.com/determined-ai/determined/pull/7963/commits/3a91652f23a23a84c3933f314bee6d29eb73d205

Reply:
i think that works